### PR TITLE
Add MediaPipeBodySegmentation implementation to video filter in the demo

### PR DESCRIPTION
**Pull request by ltrung**
**Description of changes:**
Add MediaPipeBodySegmentation implementation to video filters in the demo. There are three options:
- Beach background image
- Blue background color
- Blur

For more information, please refer to https://github.com/tensorflow/tfjs-models/blob/master/body-segmentation/README.md

This is to replace the now deprecated option [Body-Pix](https://www.npmjs.com/package/@tensorflow-models/body-pix) that is previously available in the demo under `Segmentation` option.

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
- Join meeting in demo
- Toggle any of the new three video filter MediaPipeBodySegmentation - (Beach) | (Blue) | (Blur)
- Verify that the desired effect is applied.

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
Yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: demos/browser/app/meetingV2/meetingV2.ts [hunk 1]
```diff
@@ -182,11 +185,13 @@ const BACKGROUND_BLUR_ASSET_SPEC = (BACKGROUND_BLUR_ASSET_GROUP || BACKGROUND_BL
...
 
-type VideoFilterName = 'Emojify' | 'NoOp' | 'Segmentation' | 'Resize (9/16)' | 'CircularCut' |
+type VideoFilterName = 'Emojify' | 'NoOp' | 'Resize (9/16)' | 'CircularCut' |
  'Background Blur 10% CPU' | 'Background Blur 20% CPU' | 'Background Blur 30% CPU' | 
  'Background Blur 40% CPU' | 'Background Replacement' | 'None' | 'Background Blur 2.0 - Low' |
  'Background Blur 2.0 - Medium' | 'Background Blur 2.0 - High' | 'Background Replacement 2.0 - (Beach)' |
- 'Background Replacement 2.0 - (Blue)' | 'Background Replacement 2.0 - (Default)';
+ 'Background Replacement 2.0 - (Blue)' | 'Background Replacement 2.0 - (Default)' |
+  'MediaPipeBodySegmentation - (Beach)' | 'MediaPipeBodySegmentation - (Blue)' |
+  'MediaPipeBodySegmentation - (Blur)';
```
Comment (hensmi-amazon) [inline demos/browser/app/meetingV2/meetingV2.ts:194]: Maybe we can just have one background for both 2.0 and media pipe :). Also i think we can remove my beloved emojify filter.

On a side note, should we fully deprecate v1? I can ask internally but it seems odd to still have two options 2 years after releasing v2.

File: demos/browser/app/meetingV2/video/filters/MediaPipeBodySegmentationProcessor.ts [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+import { BackgroundImageEncoding } from '../../util/BackgroundImage';
+import { BodySegmenter } from '@tensorflow-models/body-segmentation/dist/body_segmenter';
+import {
+  Segmentation
+} from '@tensorflow-models/body-segmentation/dist/shared/calculators/interfaces/common_interfaces';
+
+const segmenterConfig: bodySegmentation.MediaPipeSelfieSegmentationMediaPipeModelConfig = {
+  runtime: 'mediapipe', // or 'tfjs'
+  solutionPath: 'https://cdn.jsdelivr.net/npm/@mediapipe/selfie_segmentation',
```
Comment (hensmi-amazon) [inline demos/browser/app/meetingV2/video/filters/MediaPipeBodySegmentationProcessor.ts:21]: Isn't there a landscape model too? I couldn't tell if it was much better but seemed more appropriate for demo use case.

File: demos/browser/app/meetingV2/video/filters/MediaPipeBodySegmentationProcessor.ts [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+}
+
+export enum SegmentationOption {
+  BLUR,
+  BLUE = 1,
+  BEACH = 2,
+}
+
+/**
+ * [[SegmentationProcessors]] loads Tensorflow BodyPix model to perform image segmentation.
```
Comment (hensmi-amazon) [inline demos/browser/app/meetingV2/video/filters/MediaPipeBodySegmentationProcessor.ts:32]: stale comment.

**Review by hensmi-amazon — DISMISSED** [review]
lgtm

File: demos/browser/app/meetingV2/video/filters/MediaPipeBodySegmentationProcessor.ts (hunk 2 above)
Comment (ltrung) [reply demos/browser/app/meetingV2/video/filters/MediaPipeBodySegmentationProcessor.ts:21]: I tried in their live demo but did not see it is any better.

File: demos/browser/app/meetingV2/meetingV2.ts (hunk 1 above)
Comment (ltrung) [reply demos/browser/app/meetingV2/meetingV2.ts:194]: The react library and others still use v1 I think :D

**Review by hensmi-amazon — APPROVED** [review]

**Pull request merged**